modin/pandas/test/test_io.py

```diff
@@ -3295,3 +3294,56 @@ def test_from_ray_dataset():
     ray_df = ray.data.from_pandas(pandas_df)
     result_df = from_ray_dataset(ray_df)
     df_equals(result_df, modin_df)
+
+
+@pytest.mark.skipif(
+    condition=Engine.get() != "Dask",
+    reason="Modin Dataframe can only be converted to a Dask Dataframe if Modin uses a Dask engine.",
+)
+@pytest.mark.filterwarnings(default_to_pandas_ignore_string)
+def test_df_to_dask():
+    index = pandas.DatetimeIndex(
+        pandas.date_range("2000", freq="h", periods=len(TEST_DATA["col1"]))
+    )
+
+    modin_df, pandas_df = create_test_dfs(TEST_DATA, index=index)
+
+    dask_df = modin_df.modin.to_dask()
+    df_equals(dask_df.compute(), pandas_df)
+
+
+@pytest.mark.skipif(
+    condition=Engine.get() != "Dask",
+    reason="Modin Dataframe can only be converted to a Dask Dataframe if Modin uses a Dask engine.",
+)
+@pytest.mark.filterwarnings(default_to_pandas_ignore_string)
+def test_series_to_dask():
+    index = pandas.DatetimeIndex(
+        pandas.date_range("2000", freq="h", periods=len(TEST_DATA["col1"]))
+    )
+
+    pandas_df = pandas.DataFrame(TEST_DATA, index=index)
+    pandas_s = pandas_df.iloc[0]
+    modin_s = pd.Series(pandas_s)
+
+    dask_series = modin_s.modin.to_dask()
+    pandas_s.equals(dask_series.compute())
+
+
+@pytest.mark.skipif(
+    condition=Engine.get() != "Dask",
+    reason="Dask Dataframe can only be converted to a Modin Dataframe if Modin uses a Dask engine.",
+)
+@pytest.mark.filterwarnings(default_to_pandas_ignore_string)
+def test_from_dask_dataframe():
```

Comment:
We decided to leave to_dask and from_dask only, don't we?

Reply:
Sure, the both functions changed